backend/src/api/routers/config.py

```
@@ -0,0 +1,19 @@
+
+from ...core import controller
+
+router = APIRouter(prefix="/config", tags=["config"])
+
+@router.post("/", response_class=JSONResponse)
+async def update_config(request: Request):
+    data = await request.json()
+    try:
+        result = controller.configure_change(data)
+        return JSONResponse(status_code=200, content={"status": "ok", "result": result})
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)